diff --git a/.htaccess b/.htaccess
index 7ccb6a2..a1fdca0 100644
--- a/.htaccess
+++ b/.htaccess
@@ -29,9 +29,17 @@ DirectoryIndex index.php index.html index.htm
   php_flag magic_quotes_sybase              off
   php_flag register_globals                 off
   php_flag session.auto_start               off
+  php_flag mbstring.encoding_translation    off
+  php_flag display_startup_errors           on
+  php_flag display_errors                   on
+  php_flag html_errors                      on
+  
+  php_value max_execution_time              60
+  php_value memory_limit                    256M
+  php_value post_max_size                   500M
+  php_value upload_max_filesize             500M
   php_value mbstring.http_input             pass
   php_value mbstring.http_output            pass
-  php_flag mbstring.encoding_translation    off
 </IfModule>
 
 # Requires mod_expires to be enabled.
@@ -141,3 +149,12 @@ DirectoryIndex index.php index.html index.htm
     </FilesMatch>
   </IfModule>
 </IfModule>
+
+# HTC Files #
+AddType text/x-component .htc
+
+# @font-face fonts #
+AddType font/opentype .otf
+AddType font/ttf .ttf
+AddType application/x-font-woff .woff
+AddType application/vnd.ms-fontobject .eot
